### DISCUSSION TITLE
Run QDTrack tracking part only, providing gt detections.

### DISCUSSION THREAD

**Discussion by MarcoFrancescoMerola-rgb** [ethvis/qd-track#112] [Q&A] [answered]
Hi, i am trying to use QDTrack with BDD100K dataset for MOT on validation set.
I was able to launch QDTrack using standard code and now i want to provide gt detections and focus on tracking part.
I am having an hard time understanding where to change the model code so that i can just feed it with image and detection data and get tracks in return.
Thank you

**Comment by OceanPang** [comment, marked as answer]
Change this [`det_bboxes`](https://github.com/SysCV/qdtrack/blob/b28af06c7fdb6ce99b967302c0c7e9a557d508bf/qdtrack/models/mot/qdtrack.py#L92) with your pre-defined boxes.
